### PR TITLE
minimize only on logged in state

### LINKED ISSUE #326: Start minimized
...if possible (if a valid account and identity exist)

### PR DESCRIPTION
**Pull request by nnice**
resolves #326

### CODE REVIEW

**Comment by nnice** [comment]
hm.. somethings wrong here, i would suggest to wait until i can merge with #401, i believe that some failing issues are fixed by that

**Comment by julianseeger** [comment]
```
java.lang.NullPointerException
	at javafx.scene.CssStyleHelper.updateParentTriggerStates(CssStyleHelper.java:194)
	at javafx.scene.CssStyleHelper.createStyleHelper(CssStyleHelper.java:173)
	at javafx.scene.Node.reapplyCss(Node.java:8983)
	at javafx.scene.Node.impl_reapplyCSS(Node.java:8946)
	at javafx.scene.Node.invalidatedScenes(Node.java:854)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Parent.scenesChanged(Parent.java:667)
	at javafx.scene.Node.invalidatedScenes(Node.java:853)
	at javafx.scene.Node.setScenes(Node.java:919)
	at javafx.scene.Scene$9.invalidated(Scene.java:1119)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:111)
	at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:146)
	at javafx.scene.Scene.setRoot(Scene.java:1072)
	at javafx.scene.Scene.<init>(Scene.java:347)
	at javafx.scene.Scene.<init>(Scene.java:223)
	at de.qabel.desktop.ui.AbstractGuiTest.launchNode(AbstractGuiTest.java:55)
```
@taynaps stumbled over the same problem. It seems to be a bug in javafx that is rarely triggered (even in production). But it seems to happen more often. Maybe the cause is a misuse of some javafx features or it increases with the amount of css-style. Will be hard to debug...

File: src/test/java/de/qabel/desktop/DesktopClientGuiTest.java [hunk 1]
```diff
@@ -22,7 +24,7 @@ public void setUp() throws Exception {
     public void primaryStageShowsWhenNoAccount() throws Exception {
         startKernel(kernel);
         waitForPrimaryStage();
-        AsyncUtils.assertAsync(primaryStage::isShowing);
+        waitUntil(() -> !primaryStage.isShowing());
```
Comment (julianseeger) [inline src/test/java/de/qabel/desktop/DesktopClientGuiTest.java]: "primaryStageShowsWhenNoAccount"
the tests name implies that the stage should be shown. No reason to assert that it is not showing

**Comment by nnice** [comment]
depends on #401

**Pull request closed without merging**